packages/ragbits-core/src/ragbits/core/embeddings/litellm.py

```diff
@@ -103,10 +103,9 @@ async def embed_text(self, data: list[str], options: LiteLLMEmbeddingsOptions |
             if not response.data:
                 raise EmbeddingEmptyResponseError()
 
-            outputs.embeddings = [embedding["embedding"] for embedding in response.data]
             if response.usage:
                 outputs.completion_tokens = response.usage.completion_tokens
                 outputs.prompt_tokens = response.usage.prompt_tokens
                 outputs.total_tokens = response.usage.total_tokens
 
-        return outputs.embeddings
+        return [embedding["embedding"] for embedding in response.data]
```

Comment:
why this change?

Reply:
Otherwise we have embeddings added to outputs and using cli we have it printed to the console which makes CLITraceHandler output not readable for regular user. It was the quickest way to make output more usefull.

Reply:
yeah, but still we're printing vectors from vector store operations, which is not readable :/ i think we can overcome that with optional truncating and print only first n characters of the entry as i suggested in previous comment

Reply:
Embeddings should not be stored as traces, let's keep that change and in other PR we will handle other operations. Overall we need to think about it what parameters should be kept in traces and what removed - right now we have too much stuff in there.